website/cue/reference/components/sources/base/splunk_hec.cue

```diff
@@ -24,8 +24,19 @@ base: components: sources: splunk_hec: configuration: {
 					type: bool: default: false
 				}
 				enabled: {
-					description: "Enables end-to-end acknowledgements."
-					required:    false
+					description: """
+						Whether or not end-to-end acknowledgements are enabled.
+
+						When enabled for a sink, any source connected to that sink, where the source supports
+						end-to-end acknowledgements as well, will wait for events to be acknowledged by the sink
+						before acknowledging them at the source.
+
+						Enabling or disabling acknowledgements at the sink level takes precedence over any global
+						[`acknowledgements`][global_acks] configuration.
+
+						[global_acks]: https://vector.dev/docs/reference/configuration/global-options/#acknowledgements
+						"""
+					required: false
```

Comment:
I'm curious why this one is so different than all the other sources. Did this miss a type change?

Reply:
Fixed in 9e06dd4ef. I just added an `enabled` field directly, no more `flatten`.